mail/views.py

```diff
@@ -52,3 +54,34 @@ def post(self, request, *args, **kargs):  # pylint: disable=unused-argument, no-
                 } for batch_num, resp in enumerate(mailgun_responses)
             }
         )
+
+
+class FinancialAidMailView(APIView):
+    """
+    View for sending financial aid emails to individual learners
+    """
+    authentication_classes = (authentication.SessionAuthentication, )
+    permission_classes = (permissions.IsAuthenticated, UserCanMessageLearnersPermission, UserCanEditFinancialAid)
+
+    def post(self, request, *args, **kwargs):
+        """
+        Post request to send emails to an individual learner
+        """
+        try:
+            subject = request.data.get('email_subject') or kwargs['email_subject']
```

Comment:
interesting thing to note: if that mailgun call fails for whatever reason, the app will keep chugging along as normal. for that reason and for general separation of concerns, i'm wondering if it's a better idea to move the invocation of `send_individual_email` out of the Serializer classes and invoke it after the Serializer successfully saves the instance. come to think of it, that `save()` could fail, in which case i don't think we would want to sent the email